src/gui/updater/ocupdater.cpp

```diff
@@ -355,15 +360,17 @@ void NSISUpdater::versionInfoArrived(const UpdateInfo &info)
         qCInfo(lcUpdater) << "Client is on latest version!";
         setDownloadState(UpToDate);
     } else {
-        QString url = info.downloadUrl();
+        const QString url = info.downloadUrl();
         if (url.isEmpty()) {
             showNoUrlDialog(info);
         } else {
             _targetFile = ConfigFile::configPath() + url.mid(url.lastIndexOf('/') + 1);
-            if (QFile(_targetFile).exists()) {
+            if (QFile::exists(_targetFile)) {
                 setDownloadState(DownloadComplete);
             } else {
-                QNetworkReply *reply = qnam()->get(QNetworkRequest(QUrl(url)));
+                auto request = QNetworkRequest(QUrl(url));
+                request.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
```

Comment:
Shouldn't this be the default everywhere nowadays?

Reply:
The default are manual redirects, with the flag I enable redirects and don't limit them.